sdk/avalon_sdk/ethereum/ethereum_listener.py

```diff
@@ -220,25 +229,53 @@ def __init__(self, config):
             callback(normalized_event, *kargs, **kwargs)
             self.queue.task_done()
 
-    async def start(self, eventListener, callback, *kargs, **kwargs):
+    async def sync_handler(self, *kargs, **kwargs):
+        logging.info("Started synchronous handler to handle an event")
```

Comment:
rephrase it as "waiting for event from queue". Also mention which function puts vents in the queue for better readability